src/applications/yellow-ribbon/actions/index.js

```diff
@@ -94,7 +94,6 @@ export const fetchResultsThunk = (options = {}) => async dispatch => {
     if (trackSearch) {
       recordEvent({
         event: 'view_search_results',
-        'search-dropdown-used': false,
```

Comment:
so that `yarn lint:js` passes. Mirrors [this change](https://github.com/department-of-veterans-affairs/vets-website/pull/16122/files#diff-75d719ac7d0b7e9c08b253c271261232436f76e25072d14bf1495678c4b3e1adL97)